client/store/reports/stats/resolvers.js

```diff
@@ -0,0 +1,26 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+const { apiFetch } = wp;
```

Comment:
This should be `import apiFetch from '@wordpress/api-fetch';` – we can update the list of externals in `webpack.config.js` to enable that. [We'll want to remove references to api-request too](https://github.com/woocommerce/wc-admin/blob/master/webpack.config.js#L23), as it's been deprecated